config.go

```diff
@@ -52,8 +52,6 @@ type Config struct {
 	// Maximum duration when REQueueing (for doubling of deferred requeue)
 	MaxRequeueDelay     time.Duration `opt:"max_requeue_delay" min:"0" max:"60m" default:"15m"`
 	DefaultRequeueDelay time.Duration `opt:"default_requeue_delay" min:"0" max:"60m" default:"90s"`
-	// Unit of time for calculating consumer backoff
-	BackoffMultiplier time.Duration `opt:"backoff_multiplier" min:"0" max:"60m" default:"1s"`
```

Comment:
Unfortunately we can't just remove this (backwards compatibility) - so it might make sense to just use `BackoffMultipler` as `MinBackoffDuration`?